Tests/Test_BatteryBalancing_Task.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "BatteryBalancing.h"
+#include "Tasks.h"
+
+/******************************************************************************
+ * BatteryBalancing Task Test Plan
+ * 
+ * 1. Build the BPS code for the stm32f413 for this test file (see README.md)
+ * 2. Flash the BPS (see README.md)
+ * 3. Connect the BPS UART USB to your laptop and open PuTTY (or comparable)
+ * 4. Connect the BPS test board
+ * 5. Keep all modules at half voltage except one (of your choice) and keep that at full
```

Comment:
We should also probably test if the BPS can balance multiple modules at the same time. This could be achieved by either setting one module to (at least) 50 mV less than the other modules